cgi-bin/Apache/LiveJournal.pm

```diff
@@ -1602,7 +1603,7 @@ sub mogile_fetch {
             LJ::MemCache::add( $memkey, $paths, $cache_for ) if @paths;
         }
 
-        if ( defined $paths->[0] && $paths->[0] =~ m/^http:/ ) {
+        if ( defined $paths->[0] && $paths->[0] =~ m/^https?:/ ) {
```

Comment:
(I see what you did there ;))

Can't use protocol here; it's in a different function.